dbt_copilot_helper/commands/bootstrap.py

```diff
@@ -41,6 +42,29 @@
                         "paas": str,
                         Optional("url"): str,
                         Optional("ipfilter"): bool,
+                        Optional("count"): Or(
+                            int,
+                            {  # https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/#count
+                                "range": str,  # e.g. 1-10
```

Comment:
Can we change this to be more specific? I can put `range: asdf` and it would work.